openedx/core/djangoapps/organization_api/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+URLs for the organization app.
+"""
+from django.conf.urls import patterns, url
+
+from openedx.core.djangoapps.organization_api.api import views
+
+
+ORGANIZATION_KEY_PATTERN = r"(?P<organization_key>((?![\^'\!\(\)\*\s]).)*)"
```

Comment:
Where did this regex come from? The `short_name` is a `CharField` without regex validation: https://github.com/edx/edx-organizations/blob/a57396ba20fee3b8ed45f36c38b6b0f5baaab56e/organizations/models.py#L22.